nixops_gcp/nix/common-gce-options.nix

```diff
@@ -1,16 +1,12 @@
-{ lib }:
-
+{ lib, ... }:
 with lib;
-
 {
-
   labels = mkOption {
-      default = { };
-      example = { foo = "bar"; xyzzy = "bla"; };
-      type = types.attrsOf types.str;
-      description = ''
-        A set of key/value label pairs to assign to the instance.
-      '';
+    default = { };
+    example = { foo = "bar"; xyzzy = "bla"; };
+    type = types.nullOr (types.attrsOf types.str);
```

Comment:
hm any reason this was changed?

Reply:
Not really an issue, I assumed the user might just submit a 'null' value if no labels are needed

The user might not need to know specifically that a dict is expected

Reply:
that should be covered with the default `{}` value

Reply:
reverted per https://github.com/nix-community/nixops-gce/pull/9/commits/3ce6e4492140a55b5d22ebb1be26b063afd2055e